pkgs/tools/networking/v2ray/generic.nix

```diff
@@ -44,6 +44,10 @@ in runCommand "v2ray-${version}" {
     maintainers = with lib.maintainers; [ servalcatty ];
   };
 
+  passthru.tests = {
+    simple-vmess-proxy-test = nixosTests.v2ray;
+  };
```

Comment:
Hmm... nitpicky, but seeing as you've put this in `generic.nix`, it opens the possibility that this may get called to test an _alternate_ version of `v2ray` that also uses `generic.nix` as its base, yet, as far as I can see `nixosTests.v2ray` will always just test whatever the global default `pkgs.v2ray` is.

Reply:
I'm not expecting any alternative version by calling the same `generic.nix`. I just want to separate version-related content from build instruction at the beginning, but it seems to make things complicated.
So I think I should simply merge them to keep it easy.